### PR TITLE
[ICDS Dashboard] Excluded migrated adolscent girls 15_18 in awc aggregation

### PR DESCRIPTION
**Pull request by challabeehyv**
https://app.asana.com/0/1112385193248823/1145796612945193

##### SUMMARY
[ICDS Dashboard] Excluded migrated adolscent girls 15_18 in awc aggregation

##### FEATURE FLAG
no

### CODE REVIEW

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
this would need to change once this https://github.com/dimagi/commcare-hq/pull/26833 is merged. and it should use the migration table for checking the migration of the case.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
@challabeehyv  https://github.com/dimagi/commcare-hq/pull/26833 is now merged so can you change the migrated logic to use the other table instead of personucr

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
@challabeehyv  as there is not hurry on this. it will be good to add test cases for this

**Comment by challabeehyv** [comment]
> @challabeehyv as there is not hurry on this. it will be good to add test cases for this

modified the tests to accomodate the change

File: custom/icds_reports/tests/agg_tests/test_export_data.py [hunk 1]
```diff
@@ -655,15 +655,15 @@ def test_demographics_export_with_beta(self):
                             'Number of out of school adolescent girls (11-14 years)',
                             'Number of adolescent girls 15 to 18 years old', 'Total number of households members'],
                            ['st1', 2637, 369, 1518, '24.31 %', 120, 120, 171, 171, 1227, 1227, 56, 244, 927, 26, 0,
-                            12, 859],
+                            10, 859],
```
Comment (Rohit25negi) [inline custom/icds_reports/tests/agg_tests/test_export_data.py:658]: Why is it a decrease of 2 and everywhere else a decrease of 1?

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Looks good. left two questions.

File: custom/icds_reports/tests/agg_tests/fixtures/person_cases.csv [hunk 2]
```diff
@@ -21,7 +21,7 @@ a716755e-6b9a-4860-9556-6b5b36e78bf3_2,2017-05-01 0:00:00,,s3,,,,,,,,,,,,,,,,,,,
 89d19527-b663-480e-a3ac-50cd333a20d2,2017-07-01 0:00:00,a19,s3,b2,d1,st1,2017-04-16 0:00:00,,,F,2011-10-10,,6,,1,,,,,,,sc,,,,,,,,,,,,,,,,,,,
 9ce92cf5-eb3d-451c-8f6f-c6c67d20fab4,2017-07-01 0:00:00,a19,s3,b2,d1,st1,2017-04-10 0:00:00,,,F,1995-04-10,,22,,1,,,,,,,sc,,,,,,,,,,,,,,,,,,,
 afc5cd0d-bb23-4d9b-abfa-c638df513379,2017-07-01 0:00:00,a19,s3,b2,d1,st1,2017-04-10 0:00:00,,,M,2008-04-09,,9,,1,,,,,,,sc,,,,,,,,,,,,,,,,,,,
-bc0ee624-ca8d-43fc-9bcb-df307a81e362,2017-07-01 0:00:00,a19,s3,b2,d1,st1,2017-04-09 0:00:00,,,F,2002-04-09,,15,,1,,,,,,,sc,,,,,,,,,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (challabeehyv) [reply custom/icds_reports/tests/agg_tests/fixtures/person_cases.csv]: removed it here

File: custom/icds_reports/tests/agg_tests/test_export_data.py (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/tests/agg_tests/test_export_data.py:658]: as there is no month filter. We have 6 results for '2017-04-01' and 4 for '2017-05-01'

Comment (Rohit25negi) [reply custom/icds_reports/tests/agg_tests/test_export_data.py:658]: So you mean the case you migrated was part of 2017-04-01 and 2017-05-01 both and therefore it reduced by 2?

**Review by Rohit25negi — APPROVED** [review]
Looks good. Good to go. if the answer of the question I left is yes

File: custom/icds_reports/tests/agg_tests/test_export_data.py (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/tests/agg_tests/test_export_data.py:658]: that is correct

**Pull request merged**